src/main/java/ch/wisv/areafiftylan/controller/TeamRestController.java

```diff
@@ -156,9 +155,8 @@ public Team getTeamById(@PathVariable Long teamId) {
             "and @currentUserServiceImpl.hasAnyTicket(#username)")
     @RequestMapping(method = RequestMethod.POST, value = "/{teamId}/invites")
     public ResponseEntity<?> inviteTeamMember(@PathVariable Long teamId, @RequestBody String username) {
-        TeamInviteToken teamInviteToken = teamService.inviteMember(teamId, username);
-        return createResponseEntity(HttpStatus.OK, "User " + username + " successfully invited to Team " + teamId,
-                teamInviteToken);
+        teamService.inviteMember(teamId, username);
+        return createResponseEntity(HttpStatus.OK, "User " + username + " successfully invited to Team " + teamId);
```

Comment:
It returned the token itself, which contains a lot more information then you want to return. The Front-end requests the tokens when it needs them. That uses the `TokenResponse` object, which only contains the information required.